agent/src/Factories/BrowserFactory.php

```diff
@@ -4,14 +4,12 @@
 
 use Laravel\NightwatchAgent\Browser as NightwatchBrowser;
 use Laravel\NightwatchAgent\Contracts\Browser as BrowserContract;
-use Laravel\NightwatchAgent\PackageVersionRepository;
 use React\Http\Browser as ReactBrowser;
 use React\Socket\Connector;
 
 class BrowserFactory
 {
     public function __construct(
```

Comment:
Lets remove the constructor completely if we don't need to pass anything through